src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -478,15 +478,15 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
     }
 
     fun addFunctions(funSpecs: Iterable<FunSpec>) = apply {
-      this.funSpecs += funSpecs
+      funSpecs.forEach { addFun(it) }
```

Comment:
Nope, since `forEach` expects `(FunSpec) -> Unit`, and `addFun` is `(FunSpec) -> Builder`